### PR TITLE
Toast notificaiton - sttings wallet implemented

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/screens/settings/container/settingsContainer.js [hunk 1]
```diff
@@ -74,15 +75,54 @@ class SettingsContainer extends Component {
...
+      if (!isError) dispatch(toastNotification('Succesfuly connected!'));
+    }
+
+    if (!isError) {
+      const localTime = new Date(new Date().toISOString().split('.')[0]);
+      const serverTime = new Date(serverResp.time);
+      const isAlive = localTime - serverTime < 15000;
+
+      if (!isAlive) {
+        dispatch(toastNotification('Server not available'));
```
Comment (mistikk) [inline src/screens/settings/container/settingsContainer.js]: can you put to locale file

File: src/screens/settings/container/settingsContainer.js [hunk 2]
```diff
@@ -74,15 +75,54 @@ class SettingsContainer extends Component {
...
+
+    dispatch(setApi(server));
+
+    try {
+      serverResp = await client.database.getDynamicGlobalProperties();
+    } catch (e) {
+      isError = true;
+      dispatch(toastNotification('Connection Failed!'));
+    } finally {
+      if (!isError) dispatch(toastNotification('Succesfuly connected!'));
```
Comment (mistikk) [inline src/screens/settings/container/settingsContainer.js]: can you put to locale file

File: src/screens/settings/container/settingsContainer.js [hunk 3]
```diff
@@ -74,15 +75,54 @@ class SettingsContainer extends Component {
...
+    let isError = false;
+    const client = new Client(server, { timeout: 3000 });
+
+    dispatch(setApi(server));
+
+    try {
+      serverResp = await client.database.getDynamicGlobalProperties();
+    } catch (e) {
+      isError = true;
+      dispatch(toastNotification('Connection Failed!'));
```
Comment (mistikk) [inline src/screens/settings/container/settingsContainer.js]: can you put to locale file

Comment (ue) [reply src/screens/settings/container/settingsContainer.js]: Now if I change them app goes crush becouse of language left them for now  then wait for update language pr

Comment (mistikk) [reply src/screens/settings/container/settingsContainer.js]: alright

**Pull request merged**